### PR TITLE
Add GitHub Dependabot configuration

### PR DESCRIPTION
**Pull request by lfdebrux**
I'm trying an experiment into managing library dependencies using Dependabot, for the open RFC https://docs.google.com/document/d/1i3Pz29C1M1C0G51q6P2QWSy85oolneOcFiEdwDo74oo/

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+version: 2
```
Comment (katstevens) [inline .github/dependabot.yml:1]: Can we include a comment at the top saying which files this should be checking (similar to [the other repos](https://github.com/alphagov/digitalmarketplace-scripts/blob/master/.dependabot/config.yml))?

Comment (lfdebrux) [reply .github/dependabot.yml:1]: Ah yeah, good spot

Comment (lfdebrux) [reply .github/dependabot.yml:7]: Done

**Review by katstevens — APPROVED** [review]
Let's try it 👍

**Pull request merged**